packages/gatsby-theme-docs/utils/bootstrap-theme-addons.js

```diff
@@ -0,0 +1,45 @@
+const fs = require('fs');
+const path = require('path');
+const defaultOptions = require('./default-options');
+
+function bootstrapThemeAddOnsForMarkdownComponents(themeAddOnNames) {
+  let markdownComponents = `/* THIS IS AN AUTOGENERATED FILE, DO NOT EDIT DIRECTLY */\n`;
+  themeAddOnNames.forEach((plugin, index) => {
+    markdownComponents += `import * as components${index} from '${plugin}';`;
+  });
+  markdownComponents += `\n
+export default {
+${themeAddOnNames.map((_, index) => `...components${index},`).join('\n')}
+}
+  `;
+  const addOnsFolderPath =
+    'src/@commercetools-docs/gatsby-theme-docs/overrides';
+  const markdownComponentsPath = path.join(
+    addOnsFolderPath,
+    'markdown-components.js'
+  );
+  if (!fs.existsSync(addOnsFolderPath)) {
+    fs.mkdirSync('src/@commercetools-docs');
```

Comment:
On that occasion and since we're doing a breaking change anyways:  We could pull up the node.js Version requirement from 10 to 12 in package.json.    Then you can for example use the "recursive" option of fs.mkDirSync here, it was introduced in 10.12.0. 

Or, for a less intrusive change, set it to  `>=10.13.0` which was the start of the 10.x LTS releases and the current LTS until April 2020.

Reply:
We'll do that in a separate PR